test/powershell/engine/Remoting/SSHRemotingCmdlets.Tests.ps1

```diff
@@ -55,6 +55,6 @@ Describe "SSHConnection parameter hashtable type conversions" -Tags 'Feature', '
 
     It "<testName>" -TestCases $TestCasesSSHConnection {
         param($scriptBlock)
-        { & $scriptBlock } | Should -Throw -ErrorId '2100,PSSessionOpenFailed'
+        { & $scriptBlock } | Should -Throw -ErrorId 'PSSessionOpenFailed'
```

Comment:
Revert changes in this file, PR for this is already out: https://github.com/PowerShell/PowerShell/pull/11500

Reply:
Fixed